src/Mod/Part/BasicShapes/SkipGui.py

```diff
@@ -0,0 +1,250 @@
+#! python
+# -*- coding: utf-8 -*-
+# SkipGui.py
+# 2022, by Mark Ganson <TheMarkster>
+# LGPL 2.1 or later
+
+# this file is called from c++ in Mod/Part/Gui/Command.cpp
+# from various functions when these commands are activated:
+#   Part::Loft
+#   Part::Sweep
+#   Part::Extrude
+#   Part::Revolve
+# Then if the user has preselected certain objects and subobjects
+# the Gui can be skipped and the document objects directly created
+
+# If False is returned, then the Gui dialog should be shown
+# Else if True is returned, then the dialog can be skipped
+
+import FreeCAD as App
+import FreeCADGui as Gui
+
+def getActStr(document):
+    return "App.getDocument(\'" + document.Name + "\').ActiveObject"
```

Comment:
I think a formatted string is more readable.

I didn't look at the function usage, but beware of what `activeObject` really is.

I think you don't need to escape single quotes as your string is enclosed in double quotes.

Reply:
By formatted string you mean something like:

`return f"App.getdocument('{document.Name}').ActiveObject"`

Definitely, that looks easier to read.  And I think now python 3.6 is minimum supported version, so it should be okay to use.

ActiveObject is the most recently created/edited object, I believe.  I preferred this to adding a temporary variable.